source/_docs/guides/behat.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Automate Testing with Behat
+description: Learn how to install Behat and write simple tests.
+type: guide
+tags: [moreguides]
+permalink: docs/guides/:basename/
+contributors:
+  - ataylorme
+---
+
+[Behat](){.external} is a PHP framework for automated testing. In our [blog](link-to-blog-post){.external} we've discussed the advantages of using Behat to continuously test your site. This guide demonstrates installation of Behat and [Mink](http://mink.behat.org/en/latest/){.external}, and writing simple tests.
+
+## Before You Begin
+
+This guide requires:
+
+ - A local installation of [Composer](https://getcomposer.org/){.external}
+ - A local running dev site or remote site URL to point tests to.
+ - An empty project directory. In the commands below we assume the directory `~/projects/simple-behat-test`.
+
+<!-- \% include("content/composer-updating.html")%\-->
```

Comment:
This either needs to be removed, or un-commented and `\` replaced with `{}`, depending on the status of #4533 when this is ready.